benches/sha256.rs

```diff
@@ -390,8 +547,11 @@ cfg_if::cfg_if! {
             .measurement_time(Duration::from_secs(120))
             .sample_size(10);
             targets =
-                prove_benchmarks,
-                prove_compressed_benchmarks
+                ivc_prove_benchmarks,
+                ivc_prove_compressed_benchmarks,
+                nivc_prove_benchmarks
+                // TODO: Add when compressed SNARK is implemented for SuperNova
```

Comment:
Nit: This is more functions in the same group rather than two benchmarking groups. You can pass several groups to a criterion_main! macro. The two differ in how their statistics are aggregated in graphs for example.